src/lsst/cmservice/handlers/jobs.py

```diff
@@ -285,7 +286,7 @@ async def _reset_script(
         except FileNotFoundError:  # pragma: no cover
             pass
         try:
-            os.rmdir(submit_path)
+            shutil.rmtree(submit_path)
```

Comment:
Thinking on this since last we talked -- perhaps we want to use `aioshutil` and `await` here to not block the event loop?  File system ops could be lengthy?

Reply:
Latest change wraps some filesystem ops in async `run_in_threadpool` calls which we get for "free" from FastAPI/Starlette. There may be more such instances but this ought to cover most of the script-reset pathway.